pkg/sendconfig/sendconfig.go

```diff
@@ -179,7 +179,7 @@ func onUpdateDBMode(
 		return fmt.Errorf("creating a new syncer: %w", err)
 	}
 	syncer.SilenceWarnings = true
-	_, errs := solver.Solve(nil, syncer, kongConfig.Client, kongConfig.Concurrency, false)
+	_, errs := solver.Solve(nil, syncer, kongConfig.Client, nil, kongConfig.Concurrency, false)
```

Comment:
note: done due to deck `v1.5.0` update which was not backwards compat, but only required this change.